docs/reference/rust.md

```diff
@@ -1,14 +0,0 @@
-# Rust API
-
-The DBSP computational engine at the core of Feldera is written in
-Rust.  The engine is published on [crates.io] for easy integration and
-reference from Rust projects, and Rust API details are published on
-[docs.rs].
-
-The upstream repository is [`crates/dbsp` in the Feldera repository][repo]
-on Github.  We welcome [contributions].
-
-[crates.io]: https://crates.io/crates/dbsp
```

Comment:
these links seem valuable, are they someplace?

Reply:
Added a badge for crates.io with the readme. The rest is in the contributors guide.